fdbserver/Knobs.cpp

```diff
@@ -460,7 +460,8 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs) {
 	init( IOPS_UNITS_PER_SAMPLE,                                10000 * 1000 / STORAGE_METRICS_AVERAGE_INTERVAL_PER_KSECONDS / 100 );
 	init( BANDWIDTH_UNITS_PER_SAMPLE,                           SHARD_MIN_BYTES_PER_KSEC / STORAGE_METRICS_AVERAGE_INTERVAL_PER_KSECONDS / 25 );
 	init( BYTES_READ_UNITS_PER_SAMPLE,                          100000 ); // 100K bytes
-	init( EMPTY_READ_PENALTY,                                    20 ); // 20 bytes
+	init( EMPTY_READ_PENALTY,                                   20 ); // 20 bytes
```

Comment:
Is there a reason why empty_read_penalty is 20, instead of other value?
I guess value in certain range should be a valid `EMPTY_READ_PENALTY `?

Reply:
20 bytes is just a random value chosen by me. The exact value is not super important here. Idea is to round up the cost of empty/small reads.